br/pkg/lightning/backend/local/local.go

```diff
@@ -183,9 +180,13 @@ func (f *importClientFactoryImpl) makeConn(ctx context.Context, storeID uint64)
 	)
 	switch f.compressionType {
 	case config.CompressionNone:
-	// do nothing
+		// do nothing
 	case config.CompressionGzip:
-		opts = append(opts, grpc.WithDefaultCallOptions(grpc.UseCompressor(gzip.Name)))
+		// Use custom compressor/decompressor to speed up compression/decompression.
```

Comment:
maybe we can register our gzip Compressor with the name "gzip2" or something? Not important, we can postpone it when upstream remove this API

Reply:
No, we can't. The name will be sent to the server. The server can't recognize `gzip2`.